Lib/_pyrepl/readline.py

```diff
@@ -134,7 +134,7 @@ def get_stem(self) -> str:
         return "".join(b[p + 1 : self.pos])
 
     def get_completions(self, stem: str) -> list[str]:
-        if module_completions := self.get_module_completions():
+        if (module_completions := self.get_module_completions()) is not None:
```

Comment:
Maybe it's time to get rid of the assignment expression. I think a normal assignment followed by an `if` would be more readable.
```suggestion
        module_completions = self.get_module_completions()
        if module_completions is not None:
```